src/main/java/racing/domain/MovementStrategy.java

```diff
@@ -3,9 +3,11 @@
 import java.util.Random;
 
 public class MovementStrategy {
+
   private final int MIN_VALUE;
   private final int MAX_VALUE;
-  public MovementStrategy(int minMove, int maxMove){
+
```

Comment:
```suggestion
  public MovementStrategy() {
      this(0, 9);
  }
```
기본 생성자를 추가하면 외부에서 이 객체를 사용하는 개발자 입장에서 더 편리하게 사용할 수 있지 않을까?